scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/ProgressFileReader.java

```diff
@@ -82,8 +82,13 @@ private void createProgressFile(File workingDir, String progressFileName) throws
         try {
             Files.createDirectories(progressFileDir);
 
+            progressFileDir.toFile().setExecutable(true, false);
```

Comment:
It would be safer (for preventing some race conditions) to give the permissions when files are created. Static methods `Files.createDirectories` and `Files.createFile` support a second argument for defining permissions. Below is an example:

```
Files.createDirectories(progressFileDir, 
    PosixFilePermissions.asFileAttribute(EnumSet.of(
        PosixFilePermission.OWNER_READ, PosixFilePermission.OWNER_WRITE, ...)
```